CONTRIBUTING.md

```diff
@@ -49,6 +50,21 @@ This will setup your project along with firebase emulator in a docker environmen
 
 
 
+## Husky Setup
+
+1.install Husky and lint-staged by running the following command
+```
+npm install husky lint-staged --save-dev
+```
+2. create lint-staged.config.js file in root directory and add following code
```

Comment:
Same goes here. Since you've pushed `lint-staged.config.js` file with your pr, it should be available in the root directory after merging your pr right.

Reply:
Yes, I should write husky post installation setup. I will keep in mind thank you. 😊